ref/feature/federatedRegistry/examples.adoc

```diff
@@ -11,7 +11,7 @@ User registries that are configured with the `quickStartSecurity` element cannot
 
 === Federation of custom registries
 
-You can add basic or custom registries to a custom federated repository by defining the `participatingBaseEntry` name field in the `federatedRepository` element. The participating base entry for a custom or basic registry is the `o` organization attribute set to equal the realm name of that registry (`"o=RealmName"`). The realm name for a user registry is the value that is returned from the `getRealm()` method of that registry. The participating base entry for a custom repository is one of the base entries that is returned by the `getRepositoryBaseEntries()` method for that repository.
+You can add basic or custom registries to a custom federated repository by defining the `participatingBaseEntry` name attribute in the `federatedRepository` element. The participating base entry for a custom or basic registry is the `o` organization attribute set to equal the realm name of that registry, for example, `"o=RealmName"`. The realm name for a user registry is the value that is returned from the `getRealm()` method of that registry. The participating base entry for a custom repository is one of the base entries that is returned by the `getRepositoryBaseEntries()` method for that repository.
```

Comment:
Should it just be "attribute" after `participatingBaseEntry`, rather than "name attribute"?